pkg/sdk/testint/databases_integration_test.go

```diff
@@ -582,6 +585,32 @@ func TestInt_DatabasesAlter(t *testing.T) {
 			assert.Equal(t, "", database.Comment)
 		})
 
+		t.Run(fmt.Sprintf("Database: %s - setting and unsetting tags", testCase.DatabaseType), func(t *testing.T) {
+			databaseTest, databaseTestCleanup := testCase.CreateFn(t)
+			t.Cleanup(databaseTestCleanup)
+
+			err := client.Databases.Alter(ctx, databaseTest.ID(), &sdk.AlterDatabaseOptions{
+				SetTag: []sdk.TagAssociation{
+					{
+						Name:  tagTest.ID(),
+						Value: "v1",
+					},
+				},
+			})
+			require.NoError(t, err)
+
+			value, err := client.SystemFunctions.GetTag(ctx, tagTest.ID(), databaseTest.ID(), sdk.ObjectTypeDatabase)
+			require.NoError(t, err)
+			assert.Equal(t, "v1", value)
+
+			err = client.Databases.Alter(ctx, databaseTest.ID(), &sdk.AlterDatabaseOptions{
+				UnsetTag: []sdk.ObjectIdentifier{
+					tagTest.ID(),
+				},
+			})
+			require.NoError(t, err)
```

Comment:
we can check if the GetTag return no tag after.

Reply:
Added.